lte/gateway/c/oai/tasks/grpc_service/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ generate_cpp_protos("${ORC8R_CPP_PROTOS}" "${PROTO_SRCS}"
   "${PROTO_HDRS}" ${ORC8R_PROTO_DIR} ${ORC8R_OUT_DIR})
 
 # SPGW
-set(SPGWSRV_LTE_CPP_PROTOS spgw_service subscriberdb policydb mobilityd)
-set(SPGWSRV_LTE_GRPC_PROTOS spgw_service)
+set(SPGWSRV_LTE_CPP_PROTOS spgw_service subscriberdb policydb mobilityd session_manager)
+set(SPGWSRV_LTE_GRPC_PROTOS spgw_service session_manager) 
```

Comment:
out of curiosity, but why is session_manager needed in this set now and it wasn't before?

Reply:
Hi correct @uri200 
We have now segregated AMF and SPGW section and currently it looks like this, I'll push the changes.
hope this makes sense ?



`# AMF
set(AMFSRV_LTE_CPP_PROTOS subscriberdb session_manager)
set(AMFSRV_LTE_GRPC_PROTOS session_manager)
generate_cpp_protos("${AMFSRV_LTE_CPP_PROTOS}" "${PROTO_SRCS}"
  "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_OUT_DIR})
generate_grpc_protos("${AMFSRV_LTE_GRPC_PROTOS}" "${PROTO_SRCS}"
  "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_OUT_DIR})

# SPGW
set(SPGWSRV_LTE_CPP_PROTOS spgw_service subscriberdb policydb mobilityd)
set(SPGWSRV_LTE_GRPC_PROTOS spgw_service)
generate_cpp_protos("${SPGWSRV_LTE_CPP_PROTOS}" "${PROTO_SRCS}"
  "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_OUT_DIR})
generate_grpc_protos("${SPGWSRV_LTE_GRPC_PROTOS}" "${PROTO_SRCS}"
  "${PROTO_HDRS}" ${LTE_PROTO_DIR} ${LTE_OUT_DIR})`